packages/visualizations/src/components/Table/Cell/Format/GeoFormat.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+    import { tick } from 'svelte';
```

Comment:
I don't see where this is used ? Probably a forgotten import.

Reply:
Yes fixed here
https://github.com/opendatasoft/ods-dataviz-sdk/pull/245/commits/e7e8a4ed09953ea970f68dcb6db8144c45897773